src/ucs/type/param.h

```diff
@@ -31,6 +31,13 @@ BEGIN_C_DECLS
              (_default))
 
 
+/**
+ * Check params on fast path cond.
+ */
+#define UCS_PARAM_CHECK_FAST_PATH(_condition) \
```

Comment:
- imo, we do not need  to have `FAST_PATH` in the name, because this macro just adds check for `ENABLE_PARAMS`, which is reflected by `PARAM` prefix.
- maybe we can also define two other macro for `likely` and `unlikely` check? E.g.
`UCS_PARAM_CHECK_LIKELY`
`UCS_PARAM_CHECK_UNLIKELY`